packages/apollo-server-caching/src/connectors/memcached.ts

```diff
@@ -0,0 +1,52 @@
+import { KeyValueCache } from '../keyValueCache';
+import Memcached from 'memcached';
+
+export default class MemcachedKeyValueCache implements KeyValueCache {
+  private client;
+  private defaultSetOptions = {
+    ttl: 300,
+    tags: [],
```

Comment:
I'm not sure if default tags make sense. But a default ttl seems really useful, so maybe we could even expose that as a public property / configuration option.